Cargo.toml

```diff
@@ -2,7 +2,7 @@
 name = "mozjs"
 description = "Rust bindings to the Mozilla SpiderMonkey JavaScript engine."
 repository = "https://github.com/servo/rust-mozjs"
-version = "0.9.3"
+version = "0.9.4"
```

Comment:
One thing I'm confused is that I've bumped the version to `0.9.4` at c16e7e2473 but GitHub didn't show it 🤔

https://github.com/servo/rust-mozjs/blob/c16e7e2473ff1203bc4510bed9d6618d14cc9d31/Cargo.toml#L5